account_asset_disposal/views/account_asset_asset_view.xml

```diff
@@ -1,54 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
-     Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
-<openerp>
-<data>
 
-<record model="ir.ui.view" id="view_account_asset_asset_form">
-    <field name="name">Add disposal fields</field>
-    <field name="model">account.asset.asset</field>
-    <field name="inherit_id" ref="account_asset.view_account_asset_asset_form"/>
-    <field name="arch" type="xml">
-        <button name="set_to_close" position="attributes">
-            <attribute name='invisible'>1</attribute>
-        </button>
-        <button name="compute_depreciation_board" position="attributes">
-            <attribute name="attrs">{'invisible':[('state', '!=', 'draft')]}</attribute>
-        </button>
-        <button name="set_to_close" position="after">
-            <button name="action_disposal"
-                    states="open,close"
-                    string="Disposal"
-                    type="object"
-                    class="oe_highlight"
-            />
-            <button name="action_disposal_undo"
-                    states="disposed"
-                    string="Undo disposal"
-                    type="object"
-                    class="oe_highlight"
-            />
-        </button>
-        <field name="purchase_date" position="after">
-            <field name="disposal_date"
-                   attrs="{'invisible': [('state', '!=', 'disposed')]}"/>
-            <field name="disposal_move_id"
-                   attrs="{'invisible': [('state', '!=', 'disposed')]}"/>
-        </field>
-    </field>
-</record>
+<odoo>
 
-<record model="ir.ui.view" id="view_account_asset_category_form">
-    <field name="name">Add loss account</field>
-    <field name="model">account.asset.category</field>
-    <field name="inherit_id" ref="account_asset.view_account_asset_category_form"/>
-    <field name="arch" type="xml">
-        <field name="account_expense_depreciation_id" position="after">
-            <field name="loss_account_id"/>
+    <record model="ir.ui.view" id="view_account_asset_asset_form">
+        <field name="name">Add disposal fields</field>
+        <field name="model">account.asset.asset</field>
+        <field name="inherit_id"
+               ref="account_asset.view_account_asset_asset_form"/>
+        <field name="arch" type="xml">
+            <button name="set_to_close" position="attributes">
+                <attribute name='invisible'>1</attribute>
+            </button>
+            <button name="compute_depreciation_board" position="attributes">
+                <attribute name="attrs">{
+                    'invisible':[('state', '!=', 'draft')]}</attribute>
+            </button>
+            <button name="set_to_close" position="after">
+                <button name="action_disposal"
```

Comment:
The new button is calling another function and is positioned in the view related to the old button.